api-reference/docstubs/api/calendar-post-calendarpermissions.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Add calendarPermissions"
+description: "Add calendarPermissions by posting to the calendarPermissions collection."
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# Add calendarPermissions
+
+Namespace: microsoft.graph
+
+Add calendarPermissions by posting to the calendarPermissions collection.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Determine scopes **|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|**TODO: Determine AppOnly scopes **|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+POST /me/messages/{messageId}/event/calendar/calendarPermissions/$ref
```

Comment:
The $ref seems wrong here.  calendarPermissions is a contained nav collection of type calendarPermission.  THere shouldn't be a $ref here.